ClientApp/viking-vault/src/Dashboard.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import {constants} from "./ConstantVariables";
+import { Redirect } from "react-router-dom";
+import UserPage from "./components/UserPage";
+import { AdminPage } from "./display_user_list/AdminPage";
+
+const baseUrl = constants.baseUrl;
+
+interface IState {
+    isAdmin: boolean | null;
+}
+
+class Dashboard extends React.Component<any, IState> {
+    state = {
+        isAdmin: null
+    };
+
+    private isUserAdmin() {
+        let token = sessionStorage.getItem("Authentication-Token");
+        if(token === null) {
+            return;
+        }
+        return fetch(baseUrl+"admin", {
+            method: "GET",
+            headers: {
+              'Accept': 'application/json',
+              'Content-Type': 'application/json',
+              'x-access-token' : token.toString()
+            }
+        }).then(response => response.json())
+        .then(result => {
+            this.setState({
+                isAdmin: result
+            })
+        });
+    }
+
+    componentDidMount() {
+        this.isUserAdmin();
+    }
+
+    render() {
+        return <div>
```

Comment:
Is the `<div>` wrapper **really** needed?
Can we replace it with `<React.Fragment>` such that there is no unnecessary wrapper around the content?